cdk/tsconfig.json

```diff
@@ -0,0 +1,36 @@
+{
+  "ts-node": {
+    "compilerOptions": {
+      "module": "CommonJS"
+    }
+  },
+  "compilerOptions": {
```

Comment:
Should we extend from [`@guardian/tsconfig`](https://www.npmjs.com/package/@guardian/tsconfig) here?